bin/resources/GameIndex.yaml

```diff
@@ -27570,7 +27570,7 @@ SLPM-65576:
   name: "Tantei Jinguuji Saburou - Kind of Blue"
   region: "NTSC-J"
 SLPM-65579:
-  name: "After School Love Beat"
+  name: "Simple 2000 Series Ultimate Vol. 27 - Houkago no Love Beat"
```

Comment:
SLPM-65579 is the original version of ``Houkago no Love Beat``, not the one of Simple series.

Reply:
Both Houkago no Love Beat and Akai Ito Simple/SuperLite 2000 versions got a new serial number on their boxes but had the same disc as original versions so both are fine.